src/MSALWrapper/JsonWebTokenExtensions.cs

```diff
@@ -50,5 +50,16 @@ public static string GetDisplayName(this JsonWebToken jwt)
             jwt.TryGetClaim("name", out Claim name);
             return name?.Value;
         }
+
+        /// <summary>
+        /// Get a user's Sid.
+        /// </summary>
+        /// <param name="jwt">The JSON web token.</param>
+        /// <returns>The <see cref="string"/>.</returns>
+        public static string GetUserSid(this JsonWebToken jwt)

```

Comment:
Is Sid something tied to a user. If so, can we name this method as just `GetSid` like we have for `GetDisplayName`? Feel free to ignore if you feel `GetUserSid` gives more context :)

Reply:
I think `User` in this context is okay to have - not all JWTs are going to be attached to a User, if User is defined as a human. I would support naming this `GetSID()` (only using Caps, if we also name the property `SID`).